src/module_tests.c

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright (C) 2015 The University of Notre Dame
+This software is distributed under the GNU General Public License.
+See the file LICENSE for details.
+*/
+
+#include "console.h"
+#include "pagetable.h"
+#include "process.h"
+
+void walk_memory() {
```

Comment:
This function tries to allocate all available memory, so when it finishes the machine will halt; doesn't sound like a good idea to make it readily accessible from the command line, unless we want a sure way to halt the machine